src/client/debug/debug.proto

```diff
@@ -21,9 +24,13 @@ message ProfileRequest {
 message BinaryRequest {
 }
 
+message ClusterIDResponse {
+    string cluster_id = 1 [(gogoproto.customname) = "ClusterID"];
+}
```

Comment:
I think this probably makes more sense in the `admin` API than the `debug` API, I don't really see the cluster id as helpful for debugging, since it doesn't let you observe anything about the operations of the cluster. I think it's more of an administrative piece of information.

Reply:
Nice, it turns out admin API already exposes the cluster ID in `InspectCluster`, so I was able to reuse that rather than creating a new gRPC endpoint.